quarkus/config-api/src/main/java/org/keycloak/config/HttpOptions.java

```diff
@@ -56,7 +56,7 @@ public enum ClientAuth {
     public static final Option HTTPS_PROTOCOLS = new OptionBuilder<>("https-protocols", String.class)
             .category(OptionCategory.HTTP)
             .description("The list of protocols to explicitly enable.")
-            .defaultValue("TLSv1.3")
+            .defaultValue("TLSv1.3,TLSv1.2")
```

Comment:
This is to mitigate https://github.com/quarkusio/quarkus/pull/34468 and basically replicate the previous behaviour where TLSv1.2 worked.

Reply:
Approved, as the current NIST SP 800-52 Rev. 2 guideline recommends to enable TLSv1.3 alongside TLSv1.2, so clients can choose the "better" protocol on their capabilities.